scripts/create-a-m-o/functions.js

```diff
@@ -0,0 +1,221 @@
+const chalk = require('chalk');
+const outdent = require('outdent');
+const fs = require('fs');
+const templateJson = require('./templates/template-package.json');
+
+const getAMOType = (type) => {
+  console.log(chalk.cyan(outdent`
+
+    Ok good, you choose to create a new component of type ${chalk.bold(type)}.
+
+    Now, tell me how your new component should be called
+
+  `));
+};
+
+const prepareName = done => (userInput) => {
+  const capitalizeFirstLetter = string => string
+    .charAt(0)
+    .toUpperCase() + string
+    .slice(1);
+  const camelCase = string => string
+    .split(/[-_ ]+/)
+    .map(capitalizeFirstLetter)
+    .join('');
+  const fileName = userInput.replace(/ /g, '-').toLowerCase();
+  const className = `AXA${camelCase(fileName)}`;
+
+  console.log(chalk.cyan(outdent`
+    Ok good, we will setup all the files for your new component
+
+    Class name: ${chalk.bold(className)}
+    Folder name: ${chalk.bold(fileName)}
+  `));
+
+  done({
+    className,
+    fileName
+  });
+};
+
+const createFiles = (store, a, m, o, done) => () => {
+  const { type, fileName, className } = store;
+  const folderMap = {
+    [a]: '10-atoms',
+    [m]: '20-molecules',
+    [o]: '30-organisms',
+  };
+
+  const titleMap = {
+    [a]: 'Atoms',
+    [m]: 'Molecules',
+    [o]: 'Organisms',
+  };
+
+  const BASE_FOLDER = `./src/components/${folderMap[type]}/${fileName}`;
+
+  let compTitle = fileName.replace(/-/g, ' ');
+  compTitle =compTitle.charAt(0).toUpperCase() + compTitle.slice(1);
+
+  if (!fs.existsSync(BASE_FOLDER)) {
+    fs.mkdirSync(`${BASE_FOLDER}`);
+  }
+
+  templateJson.name = `@axa-ch/${fileName}`;
+  templateJson.version = '0.0.0-beta.0';
```

Comment:
Should we start with beta? Fine for me, I am just asking.

Reply:
absolutly or alpha

Reply:
Ah yes, I meant alpha. Obviously not a normal version without a alpha/beta tag.